.circleci/config.yml

```diff
@@ -1174,6 +1174,53 @@ jobs:
           name: "Go mod tidy"
           command: make mod-tidy && git diff --exit-code
 
+  op-service-rethdb-tests:
+    docker:
+      - image: us-docker.pkg.dev/oplabs-tools-artifacts/images/ci-builder:latest
+    resource_class: xlarge
+    steps:
+      - checkout
+      - check-changed:
+          patterns: op-service,op-node
+      - restore_cache:
+          name: Restore Go modules cache
+          key: gomod-{{ checksum "go.sum" }}
+      - run:
+          name: Install Rust toolchain
+          command: |
+            curl https://sh.rustup.rs -sSf | bash -s -- -y
```

Comment:
We should updage `ci-builder` to have the rust toolchain + deps inside of it so we don't need to install them here

Reply:
https://github.com/ethereum-optimism/optimism/pull/7918

I think we'll still want to install nightly on-the-fly, as clippy lints etc. are frequently updated. We don't need to do this, but it's normally standard to not pin nightly. Pinning stable for longer periods of time shouldn't be an issue.